vnext/rn-get-polyfills.js

```diff
@@ -1,16 +0,0 @@
-/**
```

Comment:
This is copied as part of the JS build but was added to Git unintentionally, and wasn't previously cleaned.

Reply:
Need to make sure this is in our gitignore and also remove from rnwin32